.github/workflows/tests.yaml

```diff
@@ -30,3 +30,5 @@ jobs:
                 if: "matrix.dependencies != 'lowest'"
 
             -   run: vendor/bin/phpunit
+
+            -   run: tests/system-tests/run-tests.sh
```

Comment:
This is nesting workflow to a workflow. Now GitHub Actions can't be used to and we're left with bare bash.
It should be standalone Workflow file with isolated setup

Reply:
Not really nesting a workflow...

See: `tests/system-tests` contains subdirectories with "mini-projects". After the unit tests have passed, `bin/rector` is run on all of these (like fixtures).

`run-tests.sh` is a script to run Rector on all these directories. Like PHPUnit runs on all `*Test` classes, but this script runs the entire Rector application.

If you like, I can make those system tests a dedicated workflow, but it will 99% be like `tests.yaml` (only this shell script instead of `vendor/bin/phpunit`).

Reply:
It is nesting, as it depends on all previous steps and tests.
This must be in standalone workflow. There is no way around it.